packager/app/packager_main.cc

```diff
@@ -106,7 +106,10 @@ const char kUsage[] =
     "    The group ID for the output stream. For HLS this is used as the\n"
     "    GROUP-ID attribute for EXT-X-MEDIA.\n"
     "  - playlist_name: Required for HLS output.\n"
-    "    Name of the playlist for the stream. Usually ends with '.m3u8'.\n";
+    "    Name of the playlist for the stream. Usually ends with '.m3u8'.\n"
+    "  - skip_encryption: Optional value which specifies if encryption of\n"
+    "    the stream shall be skipped. By default, if encryption is enabled,\n"
```

Comment:
Yes, good point! I'll have a go at fixing it.

I'm actually considering if one should allow setting skip_encryption=true, since it perhaps reads a bit better. Let me know if you have any particular preference here!

I also have to mention: Thank you for the friendly and good feedback I've gotten from you during the last few days! :) I personally find it very encouraging!